lib/assets.js

```diff
@@ -69,9 +69,10 @@ TradeOfferManager.prototype._addDescriptions = function(items, callback) {
 	this._requestDescriptions(descriptionRequired, (err) => {
 		if (err) {
 			callback(err);
-		} else {
-			callback(null, this._mapItemsToDescriptions(null, null, items));
+			return;
```

Comment:
Yeah, I found it strange and after reviewing the code once again I found that L152 is the only place `return callback();` is used. 

I'll split that.